tests/lib/rules/object-curly-newline.js

```diff
@@ -0,0 +1,667 @@
+/**
+ * @fileoverview Tests for object-curly-newline rule.
+ * @author Toru Nagashima
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var rule = require("../../../lib/rules/object-curly-newline"),
+    Ruvarester = require("../../../lib/testers/rule-tester");
```

Comment:
Ugh, maybe I executed "replace all" command from "let" to "var" with case insensitive. :laughing: